client/state/imports/actions.js

```diff
@@ -220,10 +228,4 @@ export const startUpload = ( importerStatus, file ) => dispatch => {
 		.then( fromApi )
 		.then( flowRight( dispatch, finishUpload( importerId ) ) )
 		.catch( flowRight( dispatch, failUpload( importerId ) ) );
```

Comment:
Avoiding `flowRight` and similar functions generally leads to more readable code:
```js
then( importerStatus => dispatch( finishUpload( importerId, importerStatus ) ) )
```